tests/system/demos/sklearn/test_sklearn.py

```diff
@@ -52,9 +52,9 @@ def create_demo_project(self) -> mlrun.projects.MlrunProject:
         self._logger.debug("Setting project functions")
         demo_project.set_function(iris_generator_function)
         demo_project.set_function("hub://describe", "describe")
-        demo_project.set_function("hub://auto_trainer", "auto_trainer")
+        demo_project.set_function("hub://auto-trainer", "auto-trainer")
         demo_project.set_function("hub://model_server", "serving")
-        demo_project.set_function("hub://model_server_tester", "live_tester")
+        demo_project.set_function("hub://model_server_tester", "live-tester")
```

Comment:
seems like part of the hub method you changed to use with `-` and some are left with `_` is this on purpose?